### PR TITLE
Java11

### PR DESCRIPTION
**Pull request by geofjamg**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
feature

**What is the current behavior?** *(You can also link to an open issue here)*
PowSyBl rely on Java8.

**What is the new behavior (if this is a feature change)?**
We only support Java11

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Comment by mathbagu** [comment]
This PR is OK for me, but we shouldn't merge it before we are also ready to switch to java 11 in other repositories. If we want to merge it now, you should revert the change about the JDK9+ profile

**Review by mathbagu — APPROVED** [review]

**Comment by geofjamg** [comment]
> 
> 
> This PR is OK for me, but we shouldn't merge it before we are also ready to switch to java 11 in other repositories. If we want to merge it now, you should revert the change about the JDK9+ profile
> 
> We also have to fix the documentation page about groovy scripting in the website.

Yes the idea is to build in Java11 with Java11 compiled core dependencies all powsybl projects (including internal one) and check everything is ok before merging this one.

**Comment by geofjamg** [comment]
> 
> 
> > This PR is OK for me, but we shouldn't merge it before we are also ready to switch to java 11 in other repositories. If we want to merge it now, you should revert the change about the JDK9+ profile
> > We also have to fix the documentation page about groovy scripting in the website.
> 
> Yes the idea is to build in Java11 (with core dependencies compiled in Java11) all powsybl projects (including internal one) and check everything is ok before merging this one.

Help needed for that!!!!

**Pull request merged**
